src/test/java/org/folio/moduserstest/GroupIT.java

```diff
@@ -126,6 +132,71 @@ public static void tearDown() {
     future.join();
   }
 
+  @Test
+  public void deletingGroupDeletesItsPatronBlockLimits(TestContext context) throws Exception {
+    /*
+      create a new patron group
+     */
```

Comment:
Can these be a one line //-style comments?

Reply:
Sure, but IMHO block comments here offer better visual separation of the test into logical steps.

Reply:
You can keep them, but I personally would just add empty lines for visual separation. Maybe it's just my prejudice agains /**/-style comments, but they don't nest and sometimes it's causing inconvenience. For example, if you try now to comment out this whole method with /**/ there'll be errors.